frontend/dynamic-demo-plugin/webpack.config.ts

```diff
@@ -2,7 +2,7 @@
 
 import * as webpack from 'webpack';
 import * as path from 'path';
-import { ConsoleRemotePlugin } from '@openshift-console/dynamic-plugin-sdk/lib/index-webpack';
+import { ConsoleRemotePlugin } from '@openshift-console/dynamic-plugin-sdk-webpack';
```

Comment:
I'll update the webpack config code snippet in the README file to not assume TypeScript usage.

In case of dynamic demo plugin, I'd keep the existing ESmodules import style since this is a `.ts` file pre-processed by `ts-node` and I'd prefer having a consistent style of module import statements. (Unless we want to make it a `.js` file and drop `ts-node` in which case we can move all statements to CommonJS `require` module import style.)